tasks/pkg-windows.yml

```diff
@@ -4,6 +4,12 @@
     msg: "The Datadog ansible role does not currently support Agent 5"
   when: datadog_agent5
 
+- name: Run 6.14.0/1 PowerShell fix
+  win_shell: |
+    Set-ExecutionPolicy Bypass -Scope Process -Force
```

Comment:
does this need the (conditional) TLS settings)?

Reply:
Where can I find the TLS setting ? I use the line from the fix page.

Reply:
if there's an Ansible task that allows downloading a file, I think we should prefer downloading the fix script first with that ansible task, and then use `win_shell` to run it